### DISCUSSION TITLE
Unable to find the http://esp2ino.local/

### DISCUSSION THREAD

**Discussion by testventure** [elahd/esp2ino#14] [Q&A] [answered]
I seem to be succeeding with the update command: 
```
wyze_updater_win.exe update -x remote -d 2CAA8E5AD5AD -f https://github.com/elahd/esp2ino/releases/latest/download/esp2ino.bin
INFO:root:Trying saved credentials from .tokens.
INFO:root:Downloading firmware to generate MD5 hash.
INFO:root:Checking device, mac=2CAA8E5AD5AD

Device type:      Light (WLPA19)
Device name:      Back Porch Light
Firmware version: 1.2.0.273
IP Address:       192.168.1.4

Pushing firmware to this device? [y/N]:y
INFO:root:Pushing firmware URL to device.
```

But after this I can't access anything via http://esp2ino.local/ and EspTouch (android) fails.  I'm likely doing something obvious wrong.  Any help would be appreciated.

**Comment by elahd** [comment, marked as answer]
Can you still control the bulb from the Wyze app? If not, can you check your router to see if the bulb is connected to your network?

**Reply by testventure** [reply]
Well, it's not operable for a few minutes after running the update command but eventually reverts to working in Wyze.

**Reply by elahd** [reply]
Got it. It sounds like the same issue as #11, which can be fixed by using a different firmware URL.

Hold off until I release v0.4 before trying again, though. There may be some stability issues with v0.3 (see #12) that prevent esp2ino from booting. Should take another week or two to test & release.

**Reply by testventure** [reply]
Oh, glad it seems to be a real issue and not one with my brain.  I appreciate you doing this and look forward to the update.  Thanks!
